.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,4 +25,4 @@ If there are following updates in the PR, ensure to request an approval from API
 - [ ] Updating API in stable version with Breaking Change Validation errors
```

Comment:
Breaking change policy link in this section should also be changed ?

Reply:
Good catch. Updated.